doc/process/npm_management.md

```diff
@@ -0,0 +1,34 @@
+# NPM Management
+
+We have a number of modules under the Node.js Foundation including:
+
+* [node-report](https://github.com/nodejs/node-report)
+* [node-inspect](https://github.com/nodejs/node-inspect)
+* [citgm](https://github.com/nodejs/citgm)
+
+We need to make sure that we have continuity in terms of being able to publish
+and update these modules.
+
+We have decided to manage these modules as follows:
+
+* Create a user called [`nodejs-foundation`][] who we always add as one of the
+  collaborators with admin rights and for which the password is maintained by
+  the build workgroup.
+* We would then add individuals as collaborators who can also publish.
+  Generally, a module push can be done by the additional collaborators.
+* In the cases where collaborators other than `nodejs-foundation`
+  cease to be active, the build workgroup would provide continuity by using the
+  `node-foundation` user to add additional collaborators who would have the
+  ability to push the module. The `node-foundation` user could also be used to
+  remove collaborators if that was ever necessary.
```

Comment:
It might be good to add another bullet point:

> The purpose of the `node-foundation` user is not to enable Build Workgroup members to publish npm modules, that should be left to the module collaborators.

I think we need to state that explicitly, to be clear on the purpose of this.